src/crontab.c

```diff
@@ -409,8 +413,17 @@ static void list_cmd(void) {
 	/* file is open. copy to stdout, close.
 	 */
 	Set_LineNum(1)
-		while (EOF != (ch = get_char(f)))
+	while (EOF != (ch = get_char(f))){
+		if(new_line){
+			if(ch == '#' && isatty(STDOUT)){
+				printf(COMMENT_COLOR);
+			}else{
```

Comment:
The whitespacing is incorrect - put a space behind if and before the { brace and around else. Also use fputs instead of printf.

Reply:
@t8m I made two commits to fix that: https://github.com/cronie-crond/cronie/pull/39/commits/4f3882c50049265bbb37d41b3f686cd93ea41430 and https://github.com/cronie-crond/cronie/pull/39/commits/e821341faeb4ac75a5fff0fba2e72ad7a6f7297d

Are you ok with them?